pkg/operator/ceph/cluster/predicate.go

```diff
@@ -134,18 +134,19 @@ func watchControllerPredicate(rookContext *clusterd.Context) predicate.Funcs {
 					// Set the cancellation flag to stop any ongoing orchestration
 					rookContext.RequestCancelOrchestration.Set()
 
-					logger.Infof("CR has changed for %q. diff=%s", objNew.Name, diff)
+					logger.Infof("CephCluster has changed for %q. diff=%s", objNew.Name, diff)
```

Comment:
Renamed CR to CephCluster 3x in this file since we know it is a CephCluster, and there is no reason to not be clear about that.